### ISSUE TITLE
Aircrack-ng 1.0-rc2 doesn't compile on Mac OS X

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#579]
**Reported by thireus on 14 Feb 2009 21:26 UTC**

Hi,

I tried to compile Aircrack-ng 1.0-rc2 on Mac OS X 10.5.6 but it doesn't!

Aircrack-ng 1.0-rc1 works well, and the actual dev source-code version compiles too.

'''Here is what I get from the Aircrack-ng 1.0-rc2 version when trying to compile:'''

```
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=0  -Iinclude   -c -o crypto.o crypto.c
In file included from crypto.c:26:
aircrack-ng.h:230: error: syntax error before pthread_cond_t
cc1: warnings being treated as errors
aircrack-ng.h:230: warning: no semicolon at end of struct or union
aircrack-ng.h:231: warning: type defaults to int in declaration of mutex
aircrack-ng.h:231: warning: data definition has no type or storage class
gmake[*** [crypto.o](1]:) Erreur 1
```

Does anyone know what are the changes to do to make it work?

(Note: The actual dev source-code compile well, but the aircrack-ng tool doesn't work, it gives a "Bus error")

**Comment by aircrack-ng** [comment]
**Comment by bigugly on 14 Feb 2009 21:26 UTC**

find and replace "user_name" with your user name and find and replace your_password with your password make executable

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 14 Feb 2009 21:26 UTC**

Replying to [bigugly](comment:1):
> find and replace "user_name" with your user name and find and replace your_password with your password make executable

Everything is now perfect :)

Thank you so much! :)

**Comment by aircrack-ng** [comment]
**Comment by misterx on 14 Feb 2009 21:26 UTC**

This ticket has 2 issue (1 is already solved in svn). A better description is available in #581.
